source/UF/Truncations.lagda

```diff
@@ -125,53 +129,54 @@ computation rules.
      I : ∥∥ₙ-ind₂ P m g ∣ x ∣[ n ] ∣ y ∣[ n ]
        ＝ ∥∥ₙ-ind (m ∣ x ∣[ n ]) (g x) ∣ y ∣[ n ]
      I = happly
-          (∥∥ₙ-ind-comp (λ u → hlevel-closed-under-Π (P u) (m u))
+          (∥∥ₙ-ind-comp (λ u → truncated-types-closed-under-Π (P u) (m u))
            (λ x' → ∥∥ₙ-ind (m ∣ x' ∣[ n ]) (g x')) x)
           ∣ y ∣[ n ]
      II : ∥∥ₙ-ind (m ∣ x ∣[ n ]) (g x) ∣ y ∣[ n ] ＝ g x y
      II = ∥∥ₙ-ind-comp (m ∣ x ∣[ n ]) (g x) y
 
 \end{code}
 
-We characterize the first couple levels of truncation (TODO: three-hlevel is a
-groupoid).
+We characterize the first couple levels of truncation.
+
+(TODO: 1-type is a groupoid).
 
 \begin{code}
 
- zero-trunc-is-contr : {X : 𝓤 ̇ } → is-contr (∥ X ∥[ 0 ])
- zero-trunc-is-contr = ∥∥ₙ-hlevel
+ −2-trunc-is-contr : {X : 𝓤 ̇ } → is-contr (∥ X ∥[ −2 ])
+ −2-trunc-is-contr = ∥∥ₙ-is-truncated
 
- one-trunc-is-prop : {X : 𝓤 ̇ } → is-prop (∥ X ∥[ 1 ])
- one-trunc-is-prop = is-prop'-implies-is-prop ∥∥ₙ-hlevel
+ −1-trunc-is-prop : {X : 𝓤 ̇ } → is-prop (∥ X ∥[ −1 ])
+ −1-trunc-is-prop = is-prop'-implies-is-prop ∥∥ₙ-is-truncated
 
- two-trunc-is-set : {X : 𝓤 ̇ } → is-set (∥ X ∥[ 2 ])
- two-trunc-is-set {𝓤} {X} {x} {y} =
-  is-prop'-implies-is-prop (∥∥ₙ-hlevel x y)
+ 0-trunc-is-set : {X : 𝓤 ̇ } → is-set (∥ X ∥[ 0 ])
+ 0-trunc-is-set {𝓤} {X} {x} {y} =
+  is-prop'-implies-is-prop (∥∥ₙ-is-truncated x y)
 
 \end{code}
 
 We demonstrate the equivalence of one-truncation and propositional truncation:
```

Comment:
Write "1-truncation" to be consistent.

Reply:
Ahh yes. In fact it should be -1-truncation.